owncloudApp/src/main/java/com/owncloud/android/ui/activity/PassCodeActivity.kt

```diff
@@ -117,10 +117,18 @@ class PassCodeActivity : BaseActivity() {
         setTextListeners()
     }
 
+    fun dpToPx(dp: Int): Int {
+        return (dp * resources.displayMetrics.density).toInt()
```

Comment:
I think that we could just set a fixed width of 32dp in the `passcode_edit_text.xml` instead of doing it programmatically. I think that it looks good when we have 4 or 6 digits